src/pages/index.js

```diff
@@ -35,8 +37,25 @@ export default function Dashboard({ data: { articleImages, quoteImage } }) {
         </Col>
         <Social />
       </Row>
-      <SelectState />
-      <Map interactive={true} legend={true} zoomControl={true} />
+
+      <Row className={styles.mapToggleButtons}>
+        <Col xs={7}>
+          <ButtonGroup>
+            <Button variant="outline-secondary" className={styles.mapToggleButtons} active={mapView==='opo-performance'} onClick={() => setMapView('opo-performance')}>OPO Performance</Button>
+            <Button variant="outline-secondary" className={styles.mapToggleButtons} active={mapView==='congressional-investigation'} onClick={() => setMapView('congressional-investigation')}>Under Congressional Investigation</Button>
+            <Button variant="outline-secondary" className={styles.mapToggleButtons} active={mapView==='black-procurement-disparity'} onClick={() => setMapView('black-procurement-disparity')}>Black Procurement Disparities</Button>

```

Comment:
I was able to figure out that when the page initially loads, the dimensions aren't being applied to the mapContainer at all. In the inspector, I don't see the mapContainer component at all initially, so there are no dimensions being applied directly to the map view at all on initial page load. Instead, the map is enclosed by the legend <div> for some reason. However, onClick of any of the toggle map view buttons, the mapContainer style and dimensions are applied to that <div>. 

I mostly changed up the dimensions to see what was happening initially. Then,  I tried passing in the `mapDimensions` prop to the legend component, which did make the initial map view bigger but it wasn't the effect we want since the width was still off slightly. I am now thinking a fix could possibly be to somehow use the `useEffect` hook to create the behavior of the `onClick` function from the map view toggle so that the initial map view has those mapDimenstons applied to it the 'opo-performance' view.